packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -57,13 +50,13 @@ const Support = decorate([
           <Card>
             <CardHeader>{_('xoaCheck')}</CardHeader>
             <CardBlock>
-              <ActionButton btnStyle='success' disabled={COMMUNITY} handler={effects.checkXoa} icon='diagnosis'>
+              <ActionButton btnStyle='success' disabled={COMMUNITY} handler={effects.refreshXoaStatus} icon='diagnosis'>
```

Comment:
Shouldn't it be `forceRefreshXoaStatus`?

Reply:
We want the user can force refresh manually ?
I find it better to call `forceRefreshXoaStatus` only in case of upgrade. To avoid multiple api call if users are calling `xoa check` in short delay